mackup/applications/xonsh.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = xonsh
```

Comment:
Can you name it `Xonsh` in here? It's supposed to be the pretty name.

Reply:
roger.